### PR TITLE
feat(dpg): update summary comment of client

### LINKED ISSUE #2195: Add comment that uniquely identifies DPG Clients
We have a need to be able to search through the azure-sdk-for-net code base and identify which clients are generated by DPG.

### PR DESCRIPTION
**Pull request by archerzz**
Change the `/// <summary>...` contents to clearly idenfity it's a DPG client.
resolve #2195

# Description
<i>Add your description here!</i>

# Checklist

To ensure a quick review and merge, please ensure:
- [ ] The PR has a understandable title and description explaining the _why_ and _what_.
- [ ] The PR is opened in draft if not ready for review yet.
   - If opened in draft, please allocate sufficient time (24 hours) after moving out of draft for review
- [ ] The branch is recent enough to not have merge conflicts upon creation.

# Ready to Land?
- [ ] Build is completely green
   - Submissions with test failures require tracking issue and approval of a CODEOWNER
- [ ] At least one +1 review by a CODEOWNER
- [ ] All -1 reviews are confirmed resolved by the reviewer 
   - Override/Marking reviews stale must be discussed with CODEOWNERS first

### CODE REVIEW

**Comment by archerzz** [comment]
@annelo-msft  Sorry I missed the [follow-up comment from Ted](https://github.com/Azure/azure-sdk-for-net/pull/29041#issuecomment-1142799105). Can you review the latest change again? I've prepared a new preview PR for `azure-sdk-for-net`: https://github.com/Azure/azure-sdk-for-net/pull/30505 Thanks.

**Comment by archerzz** [comment]
Need to merge after the fix the broken pipeline.

**Comment by archerzz** [comment]
Ready to review, since the pipeline has been restored. @annelo-msft

File: samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 namespace Azure.AI.DocumentTranslation
 {
+    // Data plane generated client. The DocumentTranslation service client.
```
Comment (pshao25) [inline samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs:19]: Why we not put it in the `<summary>`? And why not 
```suggestion
    /// Data plane generated client. The DocumentTranslation service client.
```

Comment (archerzz) [reply samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs:19]: > Why we not put it in the `<summary>`? And why not

@pshao25 see the comments from Ted: https://github.com/Azure/azure-sdk-for-net/pull/29041#issuecomment-1142799105

**Review by lirenhe — APPROVED** [review]

**Pull request merged**